### PR TITLE
Parser for iosxe/show_ap_dot11_5ghz_summary

### LINKED ISSUE #40: 21. show ap dot11 5ghz summary
```
ewlc1# show ap dot11 5ghz summary
AP Name                           Mac Address     Slot    Admin State    Oper State    Width  Txpwr           Channel
---------------------------------------------------------------------------------------------------------------------------------
ab-1-test-4800                 5c50.1501.40e0  1       Enabled        Down          20     *1/8 (23 dBm)   (36)*                        
ab21-cap40                    5c50.15f5.e5c0  1       Enabled        Up            20     *7/8 (6 dBm)    (100)*                       
ab12-cap2                     5c50.1504.7b20  1       Enabled        Up            20     *7/8 (6 dBm)    (116)*                       
ab21-cap36                    5c50.1504.87e0  1       Enabled        Up            20     *7/8 (6 dBm)    (100)*                       
ab21-cap28                    5c50.1504.8920  1       Enabled        Up            20     *7/8 (6 dBm)    (120)*                       
ab31-cap26                    5c50.1504.8940  1       Enabled        Up            20     *6/8 (9 dBm)    (128)*                       
ab22-cap16                    5c50.1504.89a0  1       Enabled        Up            20     *7/8 (8 dBm)    (48)*                        
ab22-cap24                    5c50.1504.89e0  1       Enabled        Up            20     *7/8 (8 dBm)    (36)*                        
ab22-cap18                    5c50.1504.8ac0  1       Enabled        Up            20     *7/8 (6 dBm)    (124)*                       
ab32-cap13                    5c50.1504.8bc0  1       Enabled        Up            20     *6/8 (9 dBm)    (124)*                       

AP Name                           Mac Address     Slot    Admin State    Oper State    Width  Txpwr           Channel
---------------------------------------------------------------------------------------------------------------------------------
ab22-cap10                    5c50.1504.8be0  1       Enabled        Up            20     *6/8 (9 dBm)    (132)*                       
ab11-cap18                    5c50.1504.8c20  1       Enabled        Up            20     *7/8 (8 dBm)    (48)*                        
ab22-cap8                     5c50.1504.8c40  1       Enabled        Up            20     *7/8 (8 dBm)    (149)*                       
ab22-cap22                    5c50.1504.8ce0  1       Enabled        Up            20     *8/8 (5 dBm)    (40)*                        
ab22-cap6                     5c50.1504.8d80  1       Enabled        Up            20     *7/8 (6 dBm)    (100)*                       
ab31-cap29                    5c50.1504.8dc0  1       Enabled        Up            20     *5/8 (12 dBm)   (52)*                        
ab21-cap33                    5c50.1504.8de0  1       Enabled        Up            20     *7/8 (8 dBm)    (36)*                        
ab31-cap21                    5c50.1504.8f20  1       Enabled        Up            20     *7/8 (6 dBm)    (116)*                       
ab21-cap27                    5c50.1504.8f60  1       Enabled        Up            20     *7/8 (6 dBm)    (132)*                       
ab21-cap35                    5c50.1504.aca0  1       Enabled        Up            20     *7/8 (6 dBm)    (140)*                       

AP Name                           Mac Address     Slot    Admin State    Oper State    Width  Txpwr           Channel
---------------------------------------------------------------------------------------------------------------------------------
sjc23-2-test-9130                 5c50.1504.ae60  1       Enabled        Down          20     *1/8 (24 dBm)   (52)*                        
ab22-cap20                    5c50.1504.af00  1       Enabled        Up            20     *8/8 (3 dBm)    (60)*                        
ab22-cap9                     5c50.1504.af20  1       Enabled        Up            20     *7/8 (6 dBm)    (60)*                        
ab22-cap17                    5c50.1504.af80  1       Enabled        Up            20     *7/8 (6 dBm)    (140)*                       
ab22-cap12                    5c50.1504.afa0  1       Enabled        Up            20     *8/8 (3 dBm)    (128)*                       
ab11-cap24                    5c50.1504.b000  1       Enabled        Up            20     *7/8 (6 dBm)    (140)*                       
ab21-cap31                    5c50.1504.b040  1       Enabled        Up            20     *8/8 (3 dBm)    (140)*                       
ab21-cap30                    5c50.1504.b360  1       Enabled        Up            20     *8/8 (3 dBm)    (104)*                       
ab32-cap12                    5c50.1504.d560  1       Enabled        Up            20     *7/8 (8 dBm)    (48)*                        
ab32-cap1                     5c50.1504.d660  1       Enabled        Up            20     *6/8 (11 dBm)   (36)*                        

AP Name                           Mac Address     Slot    Admin State    Oper State    Width  Txpwr           Channel
---------------------------------------------------------------------------------------------------------------------------------
ab32-cap7                     5c50.1504.d8a0  1       Enabled        Up            20     *5/8 (12 dBm)   (100)*                       
ab31-cap27                    5c50.1505.1ac0  1       Enabled        Up            20     *6/8 (11 dBm)   (149)*                       
ab11-cap22                    5c50.1505.2fe0  1       Enabled        Up            20     *8/8 (3 dBm)    (100)*                       
ab22-cap25                    5c50.1505.3600  1       Enabled        Up            20     *8/8 (3 dBm)    (52)*                        
ab32-cap17                    5c50.1505.5040  1       Enabled        Up            20     *6/8 (11 dBm)   (36)*                        
ab22-cap2                     5c50.1505.5180  1       Enabled        Up            20     *7/8 (8 dBm)    (157)*                       
ab31-cap20                    5c50.1505.51c0  1       Enabled        Up            20     *7/8 (6 dBm)    (104)*                       
ab32-cap2                     5c50.1505.52a0  1       Enabled        Up            20     *7/8 (6 dBm)    (108)*                       
ab11-cap26                    5c50.1505.5300  1       Enabled        Up            20     *7/8 (6 dBm)    (112)*                       
ab32-cap10                    5c50.1505.5380  1       Enabled        Up            20     *7/8 (8 dBm)    (44)*                        

AP Name                           Mac Address     Slot    Admin State    Oper State    Width  Txpwr           Channel
---------------------------------------------------------------------------------------------------------------------------------
ab31-cap19                    5c50.1505.5440  1       Enabled        Up            20     *7/8 (8 dBm)    (40)*                        
ab11-cap30                    5c50.1505.5480  1       Enabled        Up            20     *7/8 (6 dBm)    (64)*                        
ab31-cap28                    5c50.1505.54c0  1       Enabled        Up            20     *6/8 (11 dBm)   (48)*                        
ab11-cap32                    5c50.1505.54e0  1       Enabled        Up            20     *7/8 (8 dBm)    (149)*                       
ab31-cap30                    5c50.1505.5520  1       Enabled        Up            20     *7/8 (6 dBm)    (120)*                       
ab12-cap11                    5c50.1505.5560  1       Enabled        Up            20     *7/8 (6 dBm)    (64)*                        
ab32-cap14                    5c50.1505.5640  1       Enabled        Up            20     *6/8 (11 dBm)   (157)*                       
ab32-cap4                     5c50.1505.5660  1       Enabled        Up            20     *6/8 (9 dBm)    (64)*                        
ab11-cap29                    5c50.1505.56c0  1       Enabled        Up            20     *6/8 (9 dBm)    (116)*                       
ab11-cap27                    5c50.1505.5700  1       Enabled        Up            20     *6/8 (9 dBm)    (64)*                        

AP Name                           Mac Address     Slot    Admin State    Oper State    Width  Txpwr           Channel
---------------------------------------------------------------------------------------------------------------------------------
ab31-cap23                    5c50.1505.5760  1       Enabled        Up            20     *6/8 (9 dBm)    (104)*                       
ab12-cap3                     5c50.1505.57c0  1       Enabled        Up            20     *7/8 (8 dBm)    (36)*                        
ab22-cap15                    5c50.1505.57e0  1       Enabled        Up            20     *7/8 (6 dBm)    (104)*                       
ab12-cap1                     5c50.1505.5820  1       Enabled        Up            20     *6/8 (9 dBm)    (132)*                       
ab11-cap19                    5c50.1505.5860  1       Enabled        Up            20     *5/8 (12 dBm)   (60)*                        
ab31-cap25                    5c50.1505.58e0  1       Enabled        Up            20     *6/8 (9 dBm)    (60)*                        
ab32-cap3                     5c50.1505.5980  1       Enabled        Up            20     *6/8 (11 dBm)   (36)*                        
ab21-cap42                    5c50.1505.5b00  1       Enabled        Up            20     *6/8 (11 dBm)   (161)*                       
ab31-cap34                    5c50.1505.5b60  1       Enabled        Up            20     *7/8 (6 dBm)    (64)*                        
ab11-cap23                    5c50.1505.5be0  1       Enabled        Up            20     *7/8 (8 dBm)    (40)*                        

AP Name                           Mac Address     Slot    Admin State    Oper State    Width  Txpwr           Channel
---------------------------------------------------------------------------------------------------------------------------------
ab31-cap22                    5c50.1505.5c00  1       Enabled        Up            20     *7/8 (6 dBm)    (56)*                        
ab32-cap6                     5c50.1505.5c20  1       Enabled        Up            20     *6/8 (11 dBm)   (161)*                       
ab31-cap32                    5c50.1505.5c60  1       Enabled        Up            20     *7/8 (6 dBm)    (136)*                       
ab22-cap23                    5c50.1505.5c80  1       Enabled        Up            20     *7/8 (6 dBm)    (112)*                       
ab22-cap1                     5c50.1505.5ce0  1       Enabled        Up            20     *7/8 (6 dBm)    (120)*                       
ab32-cap9                     5c50.1505.5d60  1       Enabled        Up            20     *7/8 (8 dBm)    (153)*                       
ab31-cap24                    5c50.1505.5e40  1       Enabled        Up            20     *7/8 (6 dBm)    (136)*                       
ab21-cap39                    5c50.1505.5f20  1       Enabled        Up            20     *7/8 (6 dBm)    (64)*                        
ab11-cap20                    5c50.1505.5f40  1       Enabled        Up            20     *7/8 (6 dBm)    (128)*                       
ab12-cap17                    5c50.1505.7ba0  1       Enabled        Up            20     *6/8 (11 dBm)   (161)*                       

AP Name                           Mac Address     Slot    Admin State    Oper State    Width  Txpwr           Channel
---------------------------------------------------------------------------------------------------------------------------------
ab32-cap5                     5c50.1505.8b40  1       Enabled        Up            20     *7/8 (6 dBm)    (116)*                       
ab22-cap11                    5c50.1506.9500  1       Enabled        Up            20     *8/8 (5 dBm)    (36)*                        
ab21-cap45                    5c50.1506.9600  1       Enabled        Up            20     *7/8 (6 dBm)    (112)*                       
ab11-cap25                    5c50.1506.9720  1       Enabled        Up            20     *7/8 (8 dBm)    (161)*                       
ab21-cap29                    5c50.1506.98c0  1       Enabled        Up            20     *7/8 (6 dBm)    (56)*                        
ab11-cap21                    5c50.1506.98e0  1       Enabled        Up            20     *7/8 (6 dBm)    (52)*                        
ab32-cap16                    5c50.1506.9b80  1       Enabled        Up            20     *7/8 (6 dBm)    (64)*                        
ab22-cap3                     5c50.1506.9c20  1       Enabled        Up            20     *7/8 (6 dBm)    (56)*                        
ab12-cap14                    5c50.1506.a960  1       Enabled        Up            20     *8/8 (3 dBm)    (60)*                        
ab12-cap12                    5c50.1506.a9a0  1       Enabled        Up            20     *8/8 (3 dBm)    (120)*                       

AP Name                           Mac Address     Slot    Admin State    Oper State    Width  Txpwr           Channel
---------------------------------------------------------------------------------------------------------------------------------
ab32-cap11                    5c50.1506.a9e0  1       Enabled        Up            20     *6/8 (9 dBm)    (100)*                       
ab32-cap8                     5c50.1506.aa40  1       Enabled        Up            20     *6/8 (9 dBm)    (140)*                       
ab12-cap10                    5c50.1506.aa60  1       Enabled        Up            20     *7/8 (6 dBm)    (124)*                       
ab12-cap5                     5c50.1506.aaa0  1       Enabled        Up            20     *6/8 (9 dBm)    (52)*                        
ab12-cap9                     5c50.1506.ab00  1       Enabled        Up            20     *8/8 (3 dBm)    (108)*                       
ab11-cap31                    5c50.1506.ac40  1       Enabled        Up            20     *6/8 (9 dBm)    (104)*                       
ab12-cap13                    5c50.1506.b660  1       Enabled        Up            20     *8/8 (3 dBm)    (132)*                       
ab32-cap18                    5c50.1506.bf20  1       Enabled        Up            20     *5/8 (12 dBm)   (128)*                       
ab12-cap4                     5c50.1506.cf40  1       Enabled        Up            20     *7/8 (6 dBm)    (136)*                       
ab31-cap31                    5c50.1506.d120  1       Enabled        Up            20     *6/8 (11 dBm)   (149)*                       

AP Name                           Mac Address     Slot    Admin State    Oper State    Width  Txpwr           Channel
---------------------------------------------------------------------------------------------------------------------------------
ab31-cap33                    5c50.1506.e2e0  1       Enabled        Up            20     *6/8 (9 dBm)    (116)*                       
ab21-cap41                    5c50.1506.e360  1       Enabled        Up            20     *5/8 (12 dBm)   (124)*                       
ab22-cap21                    5c50.1507.45c0  1       Enabled        Up            20     *8/8 (5 dBm)    (161)*                       
ab21-cap46                    5c50.1507.47c0  1       Enabled        Up            20     *7/8 (6 dBm)    (56)*                        
ab21-cap37                    5c50.1507.7c00  1       Enabled        Up            20     *7/8 (8 dBm)    (153)*                       
ab22-cap19                    5c50.1507.7da0  1       Enabled        Up            20     *8/8 (3 dBm)    (120)*                       
ab12-cap15                    5c50.1507.c000  1       Enabled        Up            20     *7/8 (6 dBm)    (56)*                        
ab22-cap14                    5c50.1507.c0a0  1       Enabled        Up            20     *7/8 (6 dBm)    (140)*                       
ab22-cap4                     5c50.1507.c1e0  1       Enabled        Up            20     *7/8 (8 dBm)    (48)*                        
ab12-cap16                    5c50.1507.d780  1       Enabled        Up            20     *7/8 (6 dBm)    (116)*                       

AP Name                           Mac Address     Slot    Admin State    Oper State    Width  Txpwr           Channel
---------------------------------------------------------------------------------------------------------------------------------
ab21-cap32                    5c50.1507.dc40  1       Enabled        Up            20     *8/8 (5 dBm)    (157)*                       
ab12-cap6                     5c50.1507.e4a0  1       Enabled        Up            20     *8/8 (3 dBm)    (104)*                       
ab21-cap26                    5c50.1507.eba0  1       Enabled        Up            20     *7/8 (8 dBm)    (161)*                       
ab21-cap38                    5c50.1507.ebc0  1       Enabled        Up            20     *7/8 (8 dBm)    (149)*                       
ab21-cap43                    5c50.1507.eda0  1       Enabled        Up            20     *7/8 (6 dBm)    (64)*                        
ab21-cap34                    5c50.1507.ee20  1       Enabled        Up            20     *8/8 (3 dBm)    (124)*                       
ab11-cap28                    5c50.1507.ee60  1       Enabled        Up            20     *6/8 (9 dBm)    (136)*                       
ab12-cap7                     5c50.1507.eec0  1       Enabled        Up            20     *8/8 (5 dBm)    (48)*                        
ab22-cap5                     5c50.1507.ef60  1       Enabled        Up            20     *8/8 (5 dBm)    (153)*                       
ab21-cap44                    5c50.1507.f180  1       Enabled        Up            20     *7/8 (6 dBm)    (132)*                       

AP Name                           Mac Address     Slot    Admin State    Oper State    Width  Txpwr           Channel
---------------------------------------------------------------------------------------------------------------------------------
ab12-cap8                     5c50.1507.f1a0  1       Enabled        Up            20     *8/8 (5 dBm)    (157)*                       
ab32-cap15                    5c50.1507.f200  1       Enabled        Up            20     *6/8 (9 dBm)    (116)*                       
ab22-cap13                    5c50.1507.f2c0  1       Enabled        Up            20     *8/8 (3 dBm)    (112)*                       

```

### PR DESCRIPTION
**Pull request by reustaquio22**

### CODE REVIEW

File: src/genie/libs/parser/iosxe/show_ap.py [hunk 1]
```diff
@@ -0,0 +1,258 @@
...
+    cli_command = 'show ap dot11 5ghz summary'
+
+    def cli(self, output=None):
+        if output is None:
+            out = self.device.execute(self.cli_command)
+        else:
+            out = output
+
+        ap_dot11_5ghz_summ_dict = {}
+        # AP Name                           Mac Address     Slot    Admin State    Oper State    Width  Txpwr           Channel
```
Comment (itdependsnetworks) [inline src/genie/libs/parser/iosxe/show_ap.py:711]: Limit to 10-20 at most.

File: src/genie/libs/parser/iosxe/show_ap.py [hunk 2]
```diff
@@ -0,0 +1,258 @@
...
+        out_filter = filter_lines(raw_output=out, remove_lines=remove_lines)
+        # Key for each capture group
+        ap_name_key = ''
+
+        for line in out_filter:
+            # ab32-cap15                    5c50.1507.f200  1       Enabled        Up            20     *6/8 (9 dBm)    (116)*
+            captured_groups = match_regex(line=line, regex_patterns=regex_patterns)
+            if captured_groups:
+                # Define the root key
+                if not ap_dot11_5ghz_summ_dict.get('ap_dot11_5ghz_summ', {}):
```
Comment (itdependsnetworks) [inline src/genie/libs/parser/iosxe/show_ap.py]: Why is this needed? Could it not be `ap_dot11_5ghz_summ_dict['ap_dot11_5ghz_summ'] = {}` before the for loop?

File: src/genie/libs/parser/iosxe/show_ap.py [hunk 3]
```diff
@@ -0,0 +1,258 @@
...
+                    ap_dot11_5ghz_summ_dict['ap_dot11_5ghz_summ'] = {}
+                for k, v in captured_groups.items():
```
Comment (itdependsnetworks) [inline src/genie/libs/parser/iosxe/show_ap.py]: Can we get in the habit of doing key, value instead of k, v?

File: src/genie/libs/parser/iosxe/show_ap.py (hunk 2 above)
Comment (reustaquio22) [reply src/genie/libs/parser/iosxe/show_ap.py]: I want to populate only if it captures something. If I return `ap_dot11_5ghz_summ_dict`, the empty test will fail if i assign this `ap_dot11_5ghz_summ_dict['ap_dot11_5ghz_summ'] = {}` before the loop.

Comment (itdependsnetworks) [reply src/genie/libs/parser/iosxe/show_ap.py]: I am not tracking, there is no optional, what conditions would one happen without the other?

File: src/genie/libs/parser/iosxe/show_ap.py (hunk 1 above)
Comment (reustaquio22) [reply src/genie/libs/parser/iosxe/show_ap.py:711]: This has been reduced.

File: src/genie/libs/parser/iosxe/show_ap.py (hunk 2 above)
Comment (reustaquio22) [reply src/genie/libs/parser/iosxe/show_ap.py]: I updated and removed the root key. There was a Cisco comment on another PR to remove if it's not technically needed. [Cisco PR#212](https://github.com/CiscoTestAutomation/genieparser/pull/212/files#r477211500)

File: src/genie/libs/parser/iosxe/show_ap.py (hunk 3 above)
Comment (reustaquio22) [reply src/genie/libs/parser/iosxe/show_ap.py]: This is updated to explicitly match per capture.

**Comment by itdependsnetworks** [comment]
convert to folder structure, then create PR

**Comment by itdependsnetworks** [comment]
PR: https://github.com/CiscoTestAutomation/genieparser/pull/252 created
